rust/datafusion/src/execution/context.rs

```diff
@@ -163,6 +164,12 @@ impl ExecutionContext {
         self.register_table(name, Rc::new(CsvFile::new(filename, schema, has_header)));
     }
 
+    /// Register a Parquet file as a table so that it can be queried from SQL
+    pub fn register_parquet(&mut self, name: &str, filename: &str) {
+        let table = ParquetTable::try_new(&filename).unwrap();
```

Comment:
Could you remove this `unwrap` and have the method return a `Result<()>` please

Reply:
Changed.